crates/store/re_protos/proto/rerun/v0/remote_store.proto

```diff
@@ -94,21 +77,30 @@ enum EncoderVersion {
 }
 
 
-// ----------------- ListRecordings -----------------
+// ----------------- QueryCatalog -----------------
 
-message ListRecordingsRequest {
+message QueryCatalogRequest {
     // define which columns should be returned / projected
     // we define a separate message to make it optional.
     // If not provided, all columns should be returned
     ColumnProjection column_projection = 1;
+    // filter out specific recordings metadata
+    CatalogFilter filter = 2;
 }
 
 message ColumnProjection {
     repeated string columns = 1;
 }
 
-message ListRecordingsResponse {
-    repeated RecordingMetadata recordings = 1;
+message CatalogFilter {
+    // filter out specific recordings
```

Comment:
Let's be explicit about if this is an inclusion or exclusion filter. My expectation would be include but "filter out" sounds like exclude.

Reply:
good point, one word makes a semantic world of difference :)